src/gsi/crc32_c.c

```diff
@@ -0,0 +1,7 @@
+#include <stdio.h>
```

Comment:
see the above comment. 
I hesitate to justify adding such a c code to the currently pure fortran GSI only for this function.